### PR TITLE
Pin installed version of node to the latest LTS

### PR DESCRIPTION
**Pull request by rlucioni**
@jibsheet this is a WIP, but I wanted to at least get a foothold on the problem. @efischer19 FYI.

### CODE REVIEW

**Comment by e0d** [comment]
http://jenkins.edx.org:8080/job/ansible-provision/8088/console

:+1:

**Comment by rlucioni** [comment]
Thanks @e0d. It doesn't look like the programs tasks were run during that build. When I went to rebuild, I noticed that the programs box wasn't checked. Is that necessary?

**Comment by rlucioni** [comment]
@e0d I'm running http://jenkins.edx.org:8080/view/All/job/ansible-provision-continuous-integration/29384/ to test these changes.

**Comment by rlucioni** [comment]
@e0d @jibsheet ping. It looks like my CI build based on this branch completed successfully. Any objections to merging this?

**Comment by rlucioni** [comment]
The most recent build of this PR failed to due issues affecting npm. I'll rebuild once that's cleared up.

File: playbooks/roles/programs/tasks/main.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@
 - name: create nodeenv
   shell: >
     creates={{ programs_nodeenv_dir }}
-    {{ programs_venv_dir }}/bin/nodeenv {{ programs_nodeenv_dir }} --prebuilt
+    {{ programs_venv_dir }}/bin/nodeenv {{ programs_nodeenv_dir }} --node=4.4.7 --prebuilt
```
Comment (jibsheet) [inline playbooks/roles/programs/tasks/main.yml]: Can we make this a lowercase var in defaults main (programs_node_version or something)?
Then you just --node={{ programs_node_version }}.

Comment (rlucioni) [inline playbooks/roles/programs/tasks/main.yml]: @jibsheet updated.

**Comment by jibsheet** [comment]
👍

**Pull request merged**